groups/cdn/s3.tf

```diff
@@ -35,26 +35,48 @@ resource "aws_s3_bucket_policy" "bucket_policy" {
   policy = data.aws_iam_policy_document.s3_cloudfront_policy.json
 }
 
-data "aws_iam_policy_document" "s3_cloudfront_policy" {
-  statement {
+/// S3 server access logging  resources ///
```

Comment:
Remove the comment here—we can separately update the internal resource names to make it more obvious which resources relate to the assets bucket and the logs bucket respectively.